### PR TITLE
Elaborating on the readme

### PR DESCRIPTION
**Pull request by westonhankins**
And explaining clearly when and where they should run the script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,29 +18,40 @@ Installed Software
...
+* atom editor
+* google chrome
+
+### Requirements
+
+The computer should:
+* have Ubuntu Desktop 16.04 or 14.04 installed and updated
+* a working internet connection
+
+We recommend at least a dual-core processor and 4GB or RAM.
```
Comment (sangyye) [inline README.md]: it should be `4GB of RAM` not `or`

**Comment by westonhankins** [comment]
Fixed! :+1:

**Comment by sangyye** [comment]
Just two things I already updated the ruby version and I added nvm and node to the ansible repo. So If you change that I'm okay with merging this. So I can add a part on developement and testing if someone wants to help ;) So looks good to me 👍

**Comment by westonhankins** [comment]
Readme updated to include `nvm` and `node`!

**Comment by sangyye** [comment]
LGTM 👍

**Pull request merged**